services/provider/proto/provider.proto

```diff
@@ -169,6 +171,9 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
+    // DEPRECATED: Contains subscription channel of provider operator for client operator to match
     string desiredClientOperatorChannel = 1;
```

Comment:
@leelavg will there be a way to get this field removed in the future?

Reply:
resolving as i got LGTM in the comment of pr

Reply:
@nb-ohad yes it's possible afair, there is a `reserved ` field which takes tag numbers and throw error if used & also tells the users about tags that shouldn't be used.